galaxy/values.yaml

```diff
@@ -132,6 +132,7 @@ extraVolumeMounts: []
 
 setupJob:
   createDatabase: true
+  runAsUser: 0
```

Comment:
`runAsUser` set to 0 naturally does not work with PSP disallowing root. But I left previous value in values YAML file in case someone needs/wants to use root. If you have PSP turned on, it won;t work and you have to change 0 to e.g. 1000